sessionctx/variable/tidb_vars.go

```diff
@@ -601,6 +604,7 @@ const (
 	DefTiDBEnableAsyncCommit            = false
 	DefTiDBEnable1PC                    = false
 	DefTiDBGuaranteeExternalConsistency = false
+	DefTiDBAnalyzeVersion               = 1
```

Comment:
Does this mean we'll use old stats by default and change it manually to enable refactored stats?

Reply:
Yeah, according to the PM, it should be turned to the newer one after we have full tested it.